mds/enums.py

```diff
@@ -173,30 +173,9 @@ def choices(enum):
     "Policy types",
     [
         ("count", pgettext_lazy("Policy type", "Fleet size")),
-        (
-            "time",
-            pgettext_lazy(
-                "Policy type",
-                "Individual limitations on time spent in one or more vehicle-states. "
-                "Rule max/min refers to number of minutes.",
-            ),
-        ),
-        (
-            "speed",
-            pgettext_lazy(
-                "Policy type",
-                "Global or local speed limits. "
-                "Rule max/min refers to miles per hour.",
-            ),
-        ),
-        (
-            "user",
-            pgettext_lazy(
-                "Policy type",
-                "Information for users, e.g. about helmet laws. "
-                "Generally can't be enforced via events and telemetry.",
-            ),
-        ),
+        ("time", pgettext_lazy("Policy type", "Time spent")),
+        ("speed", pgettext_lazy("Policy type", "Speed limit")),
+        ("user", pgettext_lazy("Policy type", "Information for users")),
```

Comment:
:+1: (good to keep it simple)

Reply:
How about `Time spent` --> `Duration` ? (I don't know if "Time spent" is an actual mds concept or just a convenient key here)

Reply:
https://github.com/openmobilityfoundation/mobility-data-specification/tree/dev/policy#rule-types